rotkehlchen/tests/utils/yearn.py

```diff
@@ -0,0 +1,44 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.assets.asset import EvmToken, UnderlyingToken
+from rotkehlchen.fval import FVal
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.types import ChainID, EvmTokenKind, Timestamp
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+    from rotkehlchen.types import ChecksumEvmAddress
+
+
+def add_yearn_v3_vault_token(
+        database: 'DBHandler',
+        address: 'ChecksumEvmAddress',
+        name: str,
```

Comment:
now that I think about it don't create this function. We have `get_or_create_evm_token` that does this too. Just provide the arguments and no evm_inquirer.